CodeStarterCamp_Week2/Step2.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 고혜지 on 1/3/24.
+//
+
+import Foundation
+
+private func generateLottoNumbers() -> Set<Int> {
+  var winningNumbers: Set<Int> = Set<Int>()
+
+  repeat {
+    winningNumbers.insert(Int.random(in: 1...45))
+  } while winningNumbers.count != 6
+  
+  return winningNumbers
+}
+
+func checkLottoResult(_ input: [Int]) -> Set<Int> {
```

Comment:
```suggestion
func checkLottoWinner(with lottoNumbers: [Int]) -> Set<Int> {
```
요런 느낌으로 파라미터를 넣어보는건 어떨까요?